tests/manage/storageclass/test_delete_storageclusterinitialization.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.resources.pvc import get_all_pvc_objs
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import ManageTest, tier4, polarion_id
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+class TestDeleteStorageClusterInitialization(ManageTest):
```

Comment:
shouldn't we have a teardown logic to re-create the StorageClusterInitialization?

Reply:
It looks like it gets recreated